src/chart.zoom.js

```diff
@@ -287,6 +287,18 @@ function positionInChartArea(chartInstance, position) {
 		(position.y >= chartInstance.chartArea.top && position.y <= chartInstance.chartArea.bottom);
 }
 
+function getXAxis(chartInstance) {
+	var scales = chartInstance.scales;
+
+	for (var scaleId in scales) {
```

Comment:
I just updated `getYAxis` in order to fix an error generated by `gulp lint`. You'll have to update this to match or else the automatic style checks will fail. https://github.com/chartjs/chartjs-plugin-zoom/commit/378ac6e040175005a7b737875f8fc414e25a657b